lib/pip.js

```diff
@@ -64,15 +74,27 @@ function installRequirements() {
     }
     if (ps.status !== 0) {
       throw new Error(ps.stderr);
-    } else if (ps.stdout.trim() === 'windows') {
-      volPath = this.servicePath.replace(/\\/g, '/').replace(/^\/mnt\//, '/');
+    }
+
+    const cliPlatform = ps.stdout.trim();
+
+    if (process.platform === 'win32') {
+      bindPath = this.servicePath.replace(/\\([^\s])/g, '/$1');
+      if (cliPlatform === 'windows') {
+        bindPath = bindPath.replace(/^\/(\w)\//i, '$1:/');
+      }
+    } else if (isWsl) {
+      bindPath = this.servicePath.replace(/^\/mnt\//, '/');
```

Comment:
Looks like this should  work in either the case where we have /mn t/c (default) or where users have linked /mnt/c to /c (as in my case). I'd be happy to try it.

Reply:
Awesome, I'd love any extra Windows testing seeing as I can't do it myself. You can test this PR before it's merged by installing with `npm i heri16/serverless-python-requirements#patch-3`

Reply:
Speaking of Windows testing, @heri16  or @nickweavers do either of you know if AppVeyor has useful[0] docker support? I'll probably set it up anyway to test windows support without docker, but it'd be great to test the docker options too.

[0] I say useful, bc while CircleCI has docker support, it cant `-v` mount volumes, making it useless for this plugin

Reply:
Sorry @dschep, I can't help with AppVeyor - haven't tried it.

Thanks for the instructions on how to install from the branch before it's merge... that's really useful. Here what happened when I tried it:

```
nickw@DESKTOP-OREBB4V:/i/ddfs-serverless/python-s3-thumbnail
$npm i heri16/serverless-python-requirements#patch-3
npm WARN python-s3-thumbnail@0.1.0 No repository field.
npm WARN python-s3-thumbnail@0.1.0 No license field.

+ serverless-python-requirements@3.0.10
added 1 package and updated 1 package in 3.556s
```

```nickw@DESKTOP-OREBB4V:/i/ddfs-serverless/python-s3-thumbnail
$sls deploy -v
Serverless: Installing required Python packages with python2.7...
Serverless: Docker Image: lambci/lambda:build-python2.7

  Reference Error ----------------------------------------

  isWsl is not defined

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           9.3.0
     Serverless Version:     1.24.1
```
Maybe not having the test in the require('is-wsl') is better since it then fails with the "not defined" message rather that what was being done in the catch, which I think was to set isWsl to false making it appear we aren't under wsl?

Reply:
Hmm. Interesting. As I mentioned to @heri16, I'm not sure the try/catch is the right way to go (the dependency isn't conditional and it should run on all platforms. But if it is necessary, I think this would avoid the above issue:
```javascript
let isWsl = false
try {
  isWsl = require('is-wsl');
} catch (e) {}
```

Reply:
Happy to  test that if you point me to a commit as before?

Reply:
you should be able to re-install with the same way I gave earlier, once this PR is updated. I'll make a new branch/pr later if @heri16 doesn't get around to it.

Reply:
Exactly the same result I'm afraid:
```
nickw@DESKTOP-OREBB4V:/i/ddfs-serverless/python-s3-thumbnail
$npm i heri16/serverless-python-requirements#patch-3
npm WARN python-s3-thumbnail@0.1.0 No repository field.
npm WARN python-s3-thumbnail@0.1.0 No license field.

+ serverless-python-requirements@3.0.10
updated 1 package in 1.032s
nickw@DESKTOP-OREBB4V:/i/ddfs-serverless/python-s3-thumbnail
$sls deploy -v
Serverless: Installing required Python packages with python2.7...
Serverless: Docker Image: lambci/lambda:build-python2.7

  Reference Error ----------------------------------------

  isWsl is not defined

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           9.3.0
     Serverless Version:     1.24.1
```

Reply:
Sorry for the confusion @nickweavers this branch hasn't been updated yet, so that's expected :stuck_out_tongue_winking_eye:

Reply:
Hi @nickweavers, apologies for the mixup. Do try again with `npm i heri16/serverless-python-requirements#patch-3`

Reply:
`replace(/^\/mnt\//, '/')` was a deliberate choice to ensure that only /mnt/ would be removed (if exist).
There are two ways of using docker from WSL. One is to install packages for the linux distro of choice within WSL. The other would be to just do `ln -s /mnt/c/Program\ Files/Docker/Docker/resources/bin/docker.exe /usr/bin/docker` or `export PATH="$PATH:/mnt/c/Program\ Files/Docker/Docker/resources/bin"` along with `ln -s docker.exe /usr/bin/docker`

Reply:
Sorry for the delay, looks like the isWsl patch worked and this thing takes a while to run now! Thanks for your efforts guys, really appreciated! There's still an error, but it looks like a user error and not a system/scripting error 👍 
```
nickw@DESKTOP-OREBB4V:/i/ddfs-serverless/python-s3-thumbnail
$sls deploy -v
Serverless: Installing required Python packages with python2.7...
Serverless: Docker Image: lambci/lambda:build-python2.7
Serverless: Linking required Python packages...
Serverless: Packaging service...
Serverless: Excluding development dependencies...
Serverless: Unlinking required Python packages...
Serverless: Creating Stack...
Serverless: Checking Stack create progress...
CloudFormation - CREATE_IN_PROGRESS - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
CloudFormation - CREATE_IN_PROGRESS - AWS::S3::Bucket - ServerlessDeploymentBucket
CloudFormation - CREATE_IN_PROGRESS - AWS::S3::Bucket - ServerlessDeploymentBucket
CloudFormation - CREATE_COMPLETE - AWS::S3::Bucket - ServerlessDeploymentBucket
CloudFormation - CREATE_COMPLETE - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
Serverless: Stack create finished...
Serverless: Uploading CloudFormation file to S3...
Serverless: Uploading artifacts...
Serverless: Uploading service .zip file to S3 (12.7 MB)...
Serverless: Validating template...
Serverless: Updating Stack...
Serverless: Checking Stack update progress...
CloudFormation - UPDATE_IN_PROGRESS - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
CloudFormation - CREATE_IN_PROGRESS - AWS::IAM::Role - IamRoleLambdaExecution
CloudFormation - CREATE_IN_PROGRESS - AWS::Logs::LogGroup - S3DashthumbnailDashgeneratorLogGroup
CloudFormation - CREATE_IN_PROGRESS - AWS::IAM::Role - IamRoleLambdaExecution
CloudFormation - CREATE_IN_PROGRESS - AWS::Logs::LogGroup - S3DashthumbnailDashgeneratorLogGroup
CloudFormation - CREATE_COMPLETE - AWS::Logs::LogGroup - S3DashthumbnailDashgeneratorLogGroup
CloudFormation - CREATE_COMPLETE - AWS::IAM::Role - IamRoleLambdaExecution
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Function - S3DashthumbnailDashgeneratorLambdaFunction
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Function - S3DashthumbnailDashgeneratorLambdaFunction
CloudFormation - CREATE_COMPLETE - AWS::Lambda::Function - S3DashthumbnailDashgeneratorLambdaFunction
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Version - S3DashthumbnailDashgeneratorLambdaVersionic9c2p8ZprSApdQzriL3OpL0Rl4pkbuT1TUITtMnrI
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Permission - S3DashthumbnailDashgeneratorLambdaPermissionStephanes3thumbnailgeneratorS3
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Version - S3DashthumbnailDashgeneratorLambdaVersionic9c2p8ZprSApdQzriL3OpL0Rl4pkbuT1TUITtMnrI
CloudFormation - CREATE_IN_PROGRESS - AWS::Lambda::Permission - S3DashthumbnailDashgeneratorLambdaPermissionStephanes3thumbnailgeneratorS3
CloudFormation - CREATE_COMPLETE - AWS::Lambda::Version - S3DashthumbnailDashgeneratorLambdaVersionic9c2p8ZprSApdQzriL3OpL0Rl4pkbuT1TUITtMnrI
CloudFormation - CREATE_COMPLETE - AWS::Lambda::Permission - S3DashthumbnailDashgeneratorLambdaPermissionStephanes3thumbnailgeneratorS3
CloudFormation - CREATE_IN_PROGRESS - AWS::S3::Bucket - S3BucketStephanes3thumbnailgenerator
CloudFormation - CREATE_FAILED - AWS::S3::Bucket - S3BucketStephanes3thumbnailgenerator
CloudFormation - UPDATE_ROLLBACK_IN_PROGRESS - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
CloudFormation - UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
CloudFormation - DELETE_SKIPPED - AWS::Lambda::Version - S3DashthumbnailDashgeneratorLambdaVersionic9c2p8ZprSApdQzriL3OpL0Rl4pkbuT1TUITtMnrI
CloudFormation - DELETE_COMPLETE - AWS::S3::Bucket - S3BucketStephanes3thumbnailgenerator
CloudFormation - DELETE_IN_PROGRESS - AWS::Lambda::Permission - S3DashthumbnailDashgeneratorLambdaPermissionStephanes3thumbnailgeneratorS3
CloudFormation - DELETE_COMPLETE - AWS::Lambda::Permission - S3DashthumbnailDashgeneratorLambdaPermissionStephanes3thumbnailgeneratorS3
CloudFormation - DELETE_IN_PROGRESS - AWS::Lambda::Function - S3DashthumbnailDashgeneratorLambdaFunction
CloudFormation - DELETE_COMPLETE - AWS::Lambda::Function - S3DashthumbnailDashgeneratorLambdaFunction
CloudFormation - DELETE_IN_PROGRESS - AWS::Logs::LogGroup - S3DashthumbnailDashgeneratorLogGroup
CloudFormation - DELETE_IN_PROGRESS - AWS::IAM::Role - IamRoleLambdaExecution
CloudFormation - DELETE_COMPLETE - AWS::Logs::LogGroup - S3DashthumbnailDashgeneratorLogGroup
CloudFormation - DELETE_COMPLETE - AWS::IAM::Role - IamRoleLambdaExecution
CloudFormation - UPDATE_ROLLBACK_COMPLETE - AWS::CloudFormation::Stack - python-s3-thumbnail-dev
Serverless: Operation failed!

  Serverless Error ---------------------------------------

  An error occurred: S3BucketStephanes3thumbnailgenerator - stephane-s3-thumbnail-generator already exists.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           9.3.0
     Serverless Version:     1.24.1
```
I'll try deleting the bucket and rerunning, and confirm if it's all good when I've done that.

Reply:
Awesome!